benchmarks/sparse/utils.py

```diff
@@ -0,0 +1,114 @@
+import torch
+import itertools
+import functools
+import random
+import unittest
+import operator
+import numpy as np
+from collections import defaultdict
+import time
+
+# shim for torch.cuda.Event when running on cpu
+class Event(object):
+    def __init__(self, enable_timing):
+        pass
+
+    def record(self):
+        self.time = time.perf_counter()
+
+    def elapsed_time(self, end_event):
+        assert isinstance(end_event, Event)
+        return end_event.time - self.time
```

Comment:
Using `time.perf_counter()` is more straightforward than using Event, IMHO. Remove it?

Reply:
You mean use `time.perf_counter()` directly without having to use `Event`? I followed the conventions of the other benchmarks since they report that the Event class can later be used for timing GPUs too.